### PR TITLE
Closes #2796 - Update Docker image to add UI test dependencies

### LINKED ISSUE #2796: Update Docker image to add UI test dependencies
Dockerfile update to pre-load Google SDK and Flank lib to speed up UI tests

### PR DESCRIPTION
**Pull request by rpappalax**
Splitting off this Dockerfile update from PR #2597 so we can first update the Docker image to provide necessary UI test dependencies.
@pocmo, do you want me to also update the CHANGELOG?

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: automation/docker/Dockerfile [hunk 1]
```diff
@@ -68,6 +69,27 @@ RUN curl -L https://dl.google.com/android/repository/sdk-tools-linux-${ANDROID_S
...
+RUN mkdir -p /build/test-tools
+
+WORKDIR /build
+
+ENV TEST_TOOLS /build/test-tools
+ENV PATH ${PATH}:${TEST_TOOLS}:${TEST_TOOLS}/google-cloud-sdk/bin
+
+RUN curl https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${GOOGLE_SDK_VERSION}.0.0-linux-x86_64.tar.gz --output ./gcloud.tar.gz \
+    && tar -xvf ./gcloud.tar.gz -C ${TEST_TOOLS} \
+    && rm -f ${GOOGLE_SDK_DOWNLOAD} \
```
Comment (JohanLorenzo) [inline automation/docker/Dockerfile]: Nit: Undefined variable. I guess it refers to `./gcloud.tar.gz`?

Comment (rpappalax) [reply automation/docker/Dockerfile]: thanks, @JohanLorenzo,  done!  without the tar file, google cloud sdk adds 1G.

**Comment by pocmo** [comment]
> @pocmo, do you want me to also update the CHANGELOG?

No, this won't affect any consumers of AC. :)

**Comment by rpappalax** [comment]
@pocmo is this OK to (update then) merge?

**Pull request merged**